flight/PiOS/STM32/chibi_main.c

```diff
@@ -34,16 +34,18 @@
 #include "pios_thread.h"
 
 /* Local Variables */
+#if !defined(PIOS_INCLUDE_CHIBIOS)
```

Comment:
There's no non-chibios tasker on STM32.. I think all these can go away..

Reply:
If the file is named chibi_main.c it makes sense to take them out. Alternately if it were named stm32_main.c they should stay. We can fairly easily do that in future if we bring in another OS again so agree these can go.